homeassistant/components/heos/media_player.py

```diff
@@ -270,6 +339,14 @@ def device_state_attributes(self) -> dict:
             "media_source_id": self._player.now_playing_media.source_id,
             "media_station": self._player.now_playing_media.station,
             "media_type": self._player.now_playing_media.type,
+            ATTR_GROUP_MEMBERS: self._group,
+            # All players which can be used for grouping. Right now these are all other
+            # HEOS devices.
+            ATTR_GROUP_CANDIDATES: [
```

Comment:
This should be sourced from the available players in the API, not what's in HA as a`HeosMediaPlayer` instance.